firmware/hw_layer/adc/adc_subscription.cpp

```diff
@@ -95,6 +95,8 @@ static AdcSubscriptionEntry* findEntry() {
 	efiSetPadUnused(getAdcChannelBrainPin("adc unsubscribe", entry->Channel));
 #endif // EFI_PROD_CODE
 
+	entry->Sensor->unregister();
```

Comment:
no no, just do `sensor.unregister();`

Reply:
Sorry, Netflix and past midnight on east coast

Reply:
> Netflix and past midnight

it happens!